src/eventDispatcher.js

```diff
@@ -35,14 +35,15 @@ export class EventDispatcher {
      *  Triggers a given event.
      *  @param {String} eventName. 
      */
-    trigger(eventName) {
+    trigger(eventName, num) {
```

Comment:
In the future, we are going to trigger not just numbers, but strings, objects, etc. Having this in mind, please rename this parameter to something like `value` or similar.

Also, whenever you update the function parameters, you need to update its function description.